.github/workflows/github-to-codehub-push-sync.yml

```diff
@@ -0,0 +1,24 @@
+name: Codehub-sync
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'develop'
+      - 'releases/**'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: CASTIEL2 Codehub Sync
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: cniethammer/git-repo-sync@cb1067d118c0c209bcded585b5c78e581bb0dfec
```

Comment:
does it require specific hash rather than a version number?

Reply:
I think the repo does not really have a version number. This is what they have in their docs as well, so I would keep that for now. We can update it later to a version number if necessary.